src/controllers/filters/lang/Filter.cpp

```diff
@@ -81,6 +82,7 @@ ContextMap buildContextMap(const MessagePtr &m, chatterino::Channel *channel)
         {"author.badges", std::move(badges)},
         {"author.color", m->usernameColor},
         {"author.name", m->displayName},
+        {"author.id", m->id},
```

Comment:
This isn't the author's ID, it's the messages' ID:

https://github.com/Chatterino/chatterino2/blob/783c7530f3aedcdf493e508549b4082b13610cf9/src/providers/twitch/TwitchMessageBuilder.cpp#L587-L595